### ISSUE TITLE
add arm64 to speaker/deps/mpg123/config/linux 

### ISSUE THREAD

**Issue by WeihuaGu** [TooTallNate/node-speaker#184]
npm ERR! gyp info spawn args [ 'BUILDTYPE=Release', '-C', 'build' ]
npm ERR! In file included from ../deps/mpg123/src/output/alsa.c:10:
npm ERR! ../deps/mpg123/src/mpg123app.h:14:10: fatal error: config.h: No such file or directory
npm ERR!    14 | #include "config.h"
npm ERR!       |          ^~~~~~~~~~
npm ERR! compilation terminated.
npm ERR! make: *** [deps/mpg123/output.target.mk:126: Release/obj.target/output/deps/mpg123/src/output/alsa.o] Error 1
npm ERR! gyp ERR! build error

**Comment by JustAnEric** [comment]
Yeah, got the same issue. As an alternative, use `speaker-arm64` located on the NPMJS registry here: https://www.npmjs.com/package/speaker-arm64 @WeihuaGu
